Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -181,10 +182,10 @@
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Workspace" />
     <Reference Include="Microsoft.VisualStudio.Workspace.VSIntegration.Contracts" />
-    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
-      <HintPath>$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
+      <HintPath>$(PackagesPath)\Microsoft.VSSDK.BuildTools\tools\vssdk\bin\lib\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
-      <IncludeInVSIX>True</IncludeInVSIX>
+      <IncludeInVSIX>False</IncludeInVSIX>
```

Comment:
did this fix the newtonsoft warnings?

Reply:
Sorry but I don't recall. I did this since we don't need to ship it newtonsoft as it's part of VS already.